applications/xcm-tools-follow-up.md

```diff
@@ -0,0 +1,111 @@
+
+### Project Details
+
+1. Support XCM v3, support the setTopic instruction.(https://forum.polkadot.network/t/xcm-upgrade-to-v3/4690)
+   After the xcm V3 upgrade, messageId will be used as a credential for message tracking, which is different from using
+   message_hash(blake2_256(message)).
+   The xcm-tools will use a messageId as the credential for message tracking.
+
+2. Add send bridge xcm messages to bridgeHub Support
+
+    * `transfer_assets`
+
+3. Get the result after cross-chain and the block_hash (block_num) of the execution after sending xcm
+
+### Ecosystem Fit
+
+- Where and how does your project fit into the ecosystem?
+  XCM developers or users who want to use the XCM Feature. A solution is provided here. You can send and track your XCM
+  transaction
+
+- Who is your target audience (parachain/dapp/wallet/UI developers, designers, your own user base, some dapp's userbase,
+  yourself)?
+  wallet & explorer & parachain developer
+
+- What need(s) does your project meet?
+  Provide a solution for parachain and parachain interaction,between polkadot and ethereum, transactions between parachain and relaychain,
+  xcm message parse, and the xcm txn tracking. It can be easily integrated into the application of Golang or directly used as Tools.
+
+- Are there any other projects similar to yours in the Substrate / Polkadot / Kusama ecosystem?
+  No
+
+## Team :busts_in_silhouette:
+
+### Team members
+
+* gmajor
+
+### Contact
+
+* **Contact Name:** gmajor
+* **Contact Email:** gmajorencrypt@gmail.com
+* **Website:** <https://github.com/gmajor-encrypt>
+
+### Legal Structure
+
+individual
+
+### Team's experience
+
+I have many years of PHP development experience and nearly five years of blockchain development experience, familiar
+with PHP, GOLANG, PYTHON, Nodejs, Rust
+
+### Team Code Repos
+
+- https://github.com/gmajor-encrypt/php-scale-codec
+- https://github.com/gmajor-encrypt/php-substrate-api
+- https://github.com/gmajor-encrypt/scale-codec-comparator
+- https://github.com/gmajor-encrypt/sr25519-bindings
+- https://github.com/gmajor-encrypt/xcm-tools
+
+## Development Status :open_book:
+
+Not yet
+
+## Development Roadmap :nut_and_bolt:
+
+### Overview
+
+* **Total Estimated Duration:** 5 week
+* **Total Costs:** 9,000 USDC
+
+### Milestone 1
+
+* **Estimated duration:** 4 week
+* **FTE:**  1
+* **Costs:** 9,000 USDC
+
+| Number | Deliverable               | Specification                                                                                                                                                   |
+|-------:|---------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------|
+|    0a. | License                   | MIT or Apache 2.0                                                                                                                                               |
```

Comment:
Isn't xcm-tools Apache 2.0?

Reply:
Just want to make sure: are you planning to publish the tools under MIT license? Looks like you just removed mention of Apache.

Reply:
@semuelle Sorry, I always thought xcm-tools was under the MIT license. It has been modified now.